targets/playground/examples/thymio-challenge-pack-web-bridge.playground

```diff
@@ -2,14 +2,20 @@
 <aseba-playground>
 	<author name="Mobsya" email="info at mobsya dot org" />
 	<thanks name="Tungsteno Design" />
+	
+	<description lang="en">A ground board for Thymio Challenge Pack.</description>
+	<description lang="fr">Un plateau de sol pour le kit Thymio Challenge Pack.</description>
+	
+	<externalLink lang="en" type="wiki" target="en:thymiochallengepack" />
+	<externalLink lang="fr" type="wiki" target="fr:thymiochallengepack" />
 
 	<color name="white" r="1.0" g="1.0" b="1.0" />
 	<color name="orange" r="1" g="0.3" b="0" />
 
 	<world w="104.1" h="79.4" color="white" groundTexture="piste-thymio-challenge-pack.svgz" />
 	<wall x="50" y="35" l1="10" l2="5" h="5" mass="10" color="orange" angle="0.000" />
-	<robot type="thymio2" x="70" y="15" port="33333" name="Thymio" />
+	<robot type="thymio2" x="70" y="15" port="33330" name="Thymio" />
 	
-	<process command=":asebahttp -n ThymioChallenge -v  --autorestart -s 33333 tcp:localhost;33333" />
-
+	<process command=":asebamassloader --once thymio-default-behaviours.aesl tcp:localhost;33333" />
```

Comment:
So mass loader connects through http, it probably works but this is getting quite complicated...

Reply:
Why do we need asebamassloader as http can load a file by itself (through the `--aesl` or `-a` option) ?

Reply:
Use the `--aesl` option of asebahttp to load the AESL file without asebamassloader, nor remapping the Aseba target port:
```
<process command=":asebahttp --autorestart --aesl thymio-default-behaviours.aesl tcp:localhost;33333" />
```

Reply:
I tested and it is not working anymore. Probably last changes on http breaks something.

Reply:
i tested and it was not working now, probably last chages on http breaks someting.

Reply:
Ok, then the last change needs to be fixed and the logical solution (asebahttp) be used here.